Elemental/Components/AePagination/AePagination.razor

```diff
@@ -1,98 +1,120 @@
 @namespace Elemental.Components
 @inherits HtmlElement
 
-@if(Pageable != null)
+@if (Pageable != null)
 {
-  <div class="@_className" attributes="InputAttributesWithoutClass">
-    <div tabindex="0" class="@_pageLeftClass" @onclick="() => PageClicked(Pageable.Page - 1)">
-      <AeIcon Icon="arrow-left-s" />
-    </div>
-    <div tabindex="0" 
-      class="@($"ae page {(0 == Pageable.Page ? "active" : "")}")"
-      @onclick="@(async () => PageClicked(0))"
-    >
-      1
-    </div>
-    @if(Pageable.PageCount > 1)
-    {
-      if(Pageable.PageCount > _numberOfButtons + 1) 
-      {
-        @for(int i=0; i < _numberOfButtons; i++)
-        {
-          var localIndex = (_middleButton - _numberOfButtonsOnSide) + i;
-          var nuzzled = (i == 0 && localIndex > _numberOfButtonsOnSide - 1) 
-                     || (i == _numberOfButtons-1 && localIndex < Pageable.PageCount - _numberOfButtonsOnSide);
-              
-          <div tabindex="0" 
-            class="@GetPageClass(localIndex, nuzzled)"
-            @onclick="@(async () => PageClicked(localIndex, nuzzled))"
-          >
-            @if(nuzzled){
-              <AeIcon Icon="more" />
-            }
-            else {
-              @(localIndex+1)
-            }
-          </div>
-        }
-      }
-      else 
-      {
-        @for(int i=1; i<Pageable.PageCount-1; i++)
-        {
-          var localIndex = i;
-          <div tabindex="0" 
-            class="@($"ae page {(i == Pageable.Page ? "active" : "")}")"
-            @onclick="() => PageClicked(localIndex)"
-          >
-            @(i+1)
-          </div>
-        }
-      }
-      <div tabindex="0" 
-        class="@($"ae page {(Pageable.PageCount-1 == Pageable.Page ? "active" : "")}")"
-        @onclick="@(async () => PageClicked(Pageable.PageCount - 1))"
-      >
-        @Pageable.PageCount
-      </div>
-    }
-    <div tabindex="0" 
-      class="@_pageRightClass" 
-      @onclick="@(async () => PageClicked(Pageable.Page + 1))"
-    >
-      <AeIcon Icon="arrow-right-s" />
-    </div>
-  </div>
+	<div class="@_className" attributes="InputAttributesWithoutClass">
+		<div tabindex="0" class="@_pageLeftClass" @onclick="() => PageClicked(Pageable.Page - 1)">
+			@if(LeftIcon is null)
+			{
+				<AeIcon Icon="arrow-left-s" />
+			}
+			else
+			{
+			<span class="ae pagination-sider left">
+				@LeftIcon
+			</span>
```

Comment:
No need to wrap it with a `span`, just render `@LeftIcon` and let the users handle it